companion/src/modeledit/flightmodes.cpp

```diff
@@ -462,11 +469,11 @@ void FlightModePanel::fmClear()
       pswtch->setCurrentIndex(pswtch->findText(item.toString()));
       if (gvCount > 0 && (firmware->getCapability(GvarsFlightModes))) {
         for (int i=0; i<gvCount; i++) {
-          gvUse[i]->setCurrentIndex(0);
+          gvUse[i]->setCurrentIndex((phase.gvars[i] > 1024 ? (phase.gvars[i] - 1024) : phase.gvars[i]));
         }
       }
       for (int i=0; i<reCount; i++) {
-        reUse[i]->setCurrentIndex(0);
+        reUse[i]->setCurrentIndex((phase.rotaryEncoders[i] > 1024 ? (phase.rotaryEncoders[i] - 1024) : phase.rotaryEncoders[i]));
       }
```

Comment:
Just spotted a mistake in the two setCurrentIndex when not linked. Thought I tested. Will fix when decision on to merge or not of the set defaults.